IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -504,6 +513,27 @@ First implementation.
   end Longitude;
 
 equation
+  //---------------------------------------------------------------------------
+  // Evaluate time span of weather data
+  if ((timeSpan[1] >= 0.0) and (timeSpan[2] <= 31536000.0)) then // Data spans one year or less and ends no later than 31.12 24:00
```

Comment:
Moving the comment inside the if block will results in shorter lines and is a bit clearer what branch it is referring to.

Reply:
Ok, I can do this.